api/src/controllers/young/inscription.js

```diff
@@ -588,4 +588,54 @@ router.put("/representant-fromFranceConnect/:id", passport.authenticate("young",
   }
 });
 
+router.put("/consentements", passport.authenticate("young", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const young = await YoungObject.findById(req.user._id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND });
+
+    const needMoreConsent = getAge(young.birthdateAt) < 15;
+    const { error, value } = Joi.object({
+      parentConsentment1: Joi.boolean().required().valid(true),
+      parentConsentment2: Joi.boolean().required().valid(true),
+      parentConsentment3: Joi.boolean().required().valid(true),
+      parentConsentment4: Joi.boolean().required().valid(true),
+      parentConsentment5: Joi.boolean().required().valid(true),
+      parentConsentment6: Joi.alternatives().conditional("$needMoreConsent", {
+        is: Joi.boolean().valid(true).required(),
+        then: Joi.boolean().required().valid(true),
+        otherwise: Joi.isError(new Error()),
+      }),
+      parentConsentment7: Joi.boolean().required().valid(true),
+      consentment1: Joi.boolean().required().valid(true),
+      consentment2: Joi.alternatives().conditional("$needMoreConsent", {
+        is: Joi.boolean().valid(true).required(),
+        then: Joi.boolean().required().valid(true),
+        otherwise: Joi.isError(new Error()),
+      }),
+    }).validate(req.body, { context: { needMoreConsent: needMoreConsent } });
+
+    if (error) {
+      return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS });
+    }
+
+    if (!canUpdateYoungStatus({ body: value, current: young })) return res.status(403).send({ ok: false, code: ERRORS.OPERATION_UNAUTHORIZED });
+
+    const consentements = {
+      parentConsentment: "true",
+      consentment: "true",
+      inscriptionStep: STEPS.DOCUMENTS,
+    };
+
+    young.set(consentements);
+    await young.save({ fromUser: req.user });
+
+    await inscriptionCheck(consentements, young, req);
+
+    return res.status(200).send({ ok: true, data: young });
```

Comment:
```suggestion
    return res.status(200).send({ ok: true, data: serializeYoung(young) });
```

Normalement on doit le sérialiser pour pas renvoyer trop de trucs (side note:  je sais pas comment tu fais pour tenir)

Reply:
Je modifie ca ! en ce qui concerne le bouton enregistrer il doit marcher sur certaines page mais pas sur celle si ! Il faut que le refacto parce qu'elle tape dans la route put de base